packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -2,23 +2,18 @@
 title: Writing a unit test
 tags: Mocha, Chai, Enzyme, schemaform-utils
 ---
-
```

Comment:
I know all of these empty lines are probably unnecessary, but they do make it easier to read/edit the .mdx file. I'd recommend leaving them in.

Reply:
- [x] Add empty lines where it may serve to improve readability